rules/find-excellent-candidates/rule.md

```diff
@@ -1,7 +1,7 @@
 ---
 type: rule
 title: Do you know how to find excellent candidates?
-uri: rules-to-better-recruitment
```

Comment:
`rules-to-better-recruitment`

1. Please also add to the redirects list :)

Reply:
done